hosts/groups/meta/all.nix

```diff
@@ -0,0 +1,3 @@
+{...}: {
+  time.timeZone = "Etc/UTC";
```

Comment:
Shouldn't this be in `/nixosModules/common.nix`?

Reply:
I personally think it's much clearer to have all information about machines hierarchical (with increasing group levels etc) in the same folder structure.

Reply:
Okay, but that is not consistent with the current structure. If you want to change that - fine, but we should not have two inconsistent approaches in one repository

Reply:
Changed it, and I moved all things that are for the machines we deploy to hosts/groups/meta/common.nix.
The nixosModules/common.nix still retains the nix settings.